backend/dataall/modules/shares_base/services/share_logs_service.py

```diff
@@ -43,7 +57,7 @@ def get_share_logs_query(log_stream_name):
     @staticmethod
     def get_share_logs(shareUri):
         context = get_context()
-        if not ShareLogsService.check_view_log_permissions(context.username, context.groups, shareUri):
+        if not ShareLogsService.check_view_logs_permissions(context.username, context.groups, shareUri):
             raise exceptions.ResourceUnauthorized(
```

Comment:
Small enhancement: the exception in lines 61-65 is not needed. The exception is raised in check_view_logs_permissions

Reply:
There is an exception raised inside the `check_view_logs_permissions`, in case the admin-only setting is on. If it is set to enabled, it should follow the previous logic of checking of the  user belongs to dataset owner and/or stewards groups. In this case, a false is returned .

Reply:
I see where the issue is . Corrected code in the next update